src/components/Alerts/Alert.tsx

```diff
@@ -0,0 +1,46 @@
+import * as React from 'react';
+import { AlertBell, Checkmark, Close, Cross, Warning } from '../icons';
+import { AlertWrapper } from './AlertWrapper';
+import { AlertWithIdentifier, AlertType } from './Alerts';
+
+type Props = {
+  alert: AlertWithIdentifier;
+  index: number;
+  removeAlert: (id: string) => void;
+  className?: string;
+  noAutoClose?: boolean;
+};
+
+const Alert: React.FunctionComponent<Props> = ({ alert, index, removeAlert, className, noAutoClose }: Props) => {
+  const removeSelf = React.useCallback(() => {
+    removeAlert(alert.id);
+  }, [removeAlert, alert.id]);
+
+  React.useEffect(() => {
+    if (!noAutoClose) {
+      setTimeout(
+        () => {
+          removeSelf();
+        },
+        alert.type !== AlertType.Success ? 5000 : 10000,
```

Comment:
Maybe it's a good idea to also make this configurable.
This way in future implementations you can override the default time.